Site/webapp/wdkCustomization/js/client/main.js

```diff
@@ -1,22 +1,22 @@
 import { partial } from 'lodash';
 
 import { initialize } from '@veupathdb/web-common';
-import { wrapWdkDependencies } from '@veupathdb/study-data-access/lib/shared/wrapWdkDependencies';
+import { edaServiceUrl } from '@veupathdb/web-common/lib/config';
 
 import '@veupathdb/web-common/lib/styles/client.scss';
 
-import componentWrappers from './componentWrappers';
+import componentWrappers from './component-wrappers';
 import wrapStoreModules from './wrapStoreModules';
 import wrapWdkService from './wrapWdkService';
 import { wrapRoutes } from './routes';
-
+import { wrapWdkDependencies } from '@veupathdb/study-data-access/lib/shared/wrapWdkDependencies';
 import 'site/css/MicrobiomeSite.css';
 
 // Initialize the application.
 initialize({
   componentWrappers,
+  wrapWdkDependencies: partial(wrapWdkDependencies, edaServiceUrl),
```

Comment:
One defensive suggestion for accommodating legacy mbio and eda mbio:

```suggestion
  wrapWdkDependencies: partial(
    wrapWdkDependencies,
    useEda
      ? edaServiceUrl 
      : undefined
  ),
```